cmd/lakectl/cmd/input.go

```diff
@@ -5,16 +5,30 @@ import (
 	"github.com/spf13/pflag"
 )
 
-func confirm(flags *pflag.FlagSet, question string) (bool, error) {
-	force, _ := flags.GetBool("force")
-	if force {
+const (
+	AutoConfirmFlagName     = "yes"
+	AutoConfigFlagShortName = "y"
+	AutoConfirmFlagHelp     = "Automatically say yes to all confirmations"
+)
+
+func AssignAutoConfirmFlag(flags *pflag.FlagSet) {
+	flags.BoolP(AutoConfirmFlagName, AutoConfigFlagShortName, false, AutoConfirmFlagHelp)
```

Comment:
why not use global persist flag for "yes" it is very common to have it available for all commands

Reply:
That's what we had before. I don't want to do that because:
1. it will show up in the help text for all commands eventhough it's a no-op for most, so confusing
2. it's less flexible since `-y` might take a different meaning in another command but can't be used anymore

Reply:
1. The fact that the global flag always appears the chance that somebody will run it over is lower.
2. The user get used to the consistent interface
3. Any new command that we will add or change the current flow of existing comment doesn't need to worry about how auto-confirmation is done (fewer bugs)